transforms/message-remove-classic-theme/__testfixtures__/Basic.input.js

```diff
@@ -0,0 +1,11 @@
+import Message from "carbon-react/lib/components/message";
+export default () => <Message as="info" roundedCorners />;
+export const withProps = () => <Message as="info" title="Example" roundedCorners={true} />;
+const as = "info";
+const roundedCorners = true;
+export const asVariable = () => <Message as={as} roundedCorners={roundedCorners} />;
+const props = { as: "info", title: "Example", roundedCorners: true }
+export const spread = () => <Message {...props} roundedCorners />;
+export const ObjectExpressionsLiteralReplacement = () => <Message {...{ as: "info", title: "Example", roundedCorners: true }} />;
+export const ObjectExpressionsIdentifierReplacement1 = () => <Message {...{ as, title: "Example", roundedCorners }} />;
+export const ObjectExpressionsIdentifierReplacement2 = () => <Message {...{ as: as, title: "Example", roundedCorners: roundedCorners }} />;
```

Comment:
Are we bothered about these "No newline..." messages?

Reply:
They're only displayed in the github ui. It just means theres no newline at the end of the file.